CHANGELOG.rst

```diff
@@ -1,6 +1,17 @@
 Changelog
 ---------
 
+0.1.5 (2024-04-15)
+******************
+
+- Launch training using the last build configuration from a project
+- Enable non-interactive mode
+- Introduce method that checks if the project meets the minimum requirements for training
```

Comment:
Especially since this is public, to help users understand the changes, we could be more concrete (eg `Introduce method` -> ```Add ``Project.is_trainable()`` ```)